storage/inkless/src/main/resources/db/migration/V3__Add_logs_size.sql

```diff
@@ -0,0 +1,300 @@
+-- Copyright (c) 2025 Aiven, Helsinki, Finland. https://aiven.io/
+
+-- 1. Introduce the `byte_size` column.
+
+ALTER TABLE logs
+ADD COLUMN byte_size byte_size_t DEFAULT 0;
+
+UPDATE logs
+SET byte_size = aggregated.total_byte_size
+FROM (
+    SELECT topic_id, partition, SUM(byte_size) AS total_byte_size
+    FROM batches
+    GROUP BY topic_id, partition
+) AS aggregated
+WHERE logs.topic_id = aggregated.topic_id
+    AND logs.partition = aggregated.partition;
+
+-- 2. Update functions to support `byte_size`.
```

Comment:
There are two ways to affect the size of the log (not taking into account the deletion of the whole log, because in this case the very record of it is deleted from the DB):
1. Append records/batches, done by `commit_file_v1`.
2. Delete records/batches, done by `delete_records_v1`.